snippets/pulse-loader.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Pulse loader
+tags: animation
+---
+
+Creates a pulse effect loader animation using `animation-delay` property.
+
+```html
+<div class="ripple-loader">
+    <div></div>
+    <div></div>
+</div>
+```
+
+```css
+    .ripple-loader {
```

Comment:
Please fix the indentation. Top level rules should have `0` spaces and every subsequent level should be indented by `2` spaces more than the last one.